### ISSUE TITLE
Demo World on Polygon Main, but Demo Profiles on Testnet

### ISSUE THREAD

**Issue by mezamashii** [Utopia42-club/utopia42-unity#71]
You can’t add an avatar on demo https://dev.utopia42.club/game because the map is on polygon but the avatar system is still on testnet. 

[from User Testing July 28, 2022](https://docs.google.com/document/d/1wdCe8GZmm2djueDQTW5pYlzAsNtIRHc4Rpvijdz3KaY/edit?usp=sharing)

**Comment by rasoulMrz** [comment]
Note: The demo server is the multiverse version of the game, so the server is not associated with any networks! It depends on metaverses and their networks (we can have metaverses on various nets). Currently, we only have TestNet metaverses! so you will be connected to this chain.
Note: The profile system is currently managed by a Server and is independent of networks! so you will have the same profile and avatar across the chains and metaverses. However, the game asks you to connect your Metamask to the same network as your current metaverse (this could be omitted in the future).

I could not reproduce this issue. 
Procedure:

- I logged in with a new wallet.
- It asked me to edit my profile
- Asked me to connect my Metamask to PolygonTestnet
- I entered my avatar URL and saved my profile
- Everything worked fine.
